x-pack/plugins/security_solution/public/management/pages/policy/view/artifacts/empty/policy_artifacts_empty_unassigned.tsx

```diff
@@ -54,42 +49,41 @@ export const PolicyArtifactsEmptyUnassigned = memo<CommonProps>(
       [navigateCallback]
     );
     return (
-      <EuiPageTemplate template="centeredContent">
-        <EuiEmptyPrompt
-          iconType="plusInCircle"
-          data-test-subj="policy-artifacts-empty-unassigned"
-          title={<h2>{labels.emptyUnassignedTitle}</h2>}
-          body={
-            canWriteArtifact
-              ? labels.emptyUnassignedMessage(policyName)
-              : labels.emptyUnassignedNoPrivilegesMessage(policyName)
-          }
-          actions={[
-            ...(canCreateArtifactsByPolicy && canWriteArtifact
-              ? [
-                  <EuiButton
-                    color="primary"
-                    fill
-                    onClick={onClickPrimaryButtonHandler}
-                    data-test-subj="unassigned-assign-artifacts-button"
-                  >
-                    {labels.emptyUnassignedPrimaryActionButtonTitle}
-                  </EuiButton>,
-                ]
-              : []),
-            canWriteArtifact ? (
-              // eslint-disable-next-line @elastic/eui/href-or-on-click
-              <EuiLink
-                onClick={onClickHandler}
-                href={toRouteUrl}
-                data-test-subj="unassigned-manage-artifacts-button"
-              >
-                {labels.emptyUnassignedSecondaryActionButtonTitle}
-              </EuiLink>
-            ) : null,
-          ]}
-        />
-      </EuiPageTemplate>
+      <EuiEmptyPrompt
```

Comment:
Q: Is there a visual difference if we do 

```json5
<EuiPageTemplate>
 <EuiPageTemplate.EmptyPrompt>
...
...
 </EuiPageTemplate.EmptyPrompt>
</EuiPageTemplate>

Reply:
These components are actually already wrapped in an `EuiPageTemplate`, so if I push another containing template, the UI will not look correct.

You are correct in these comments, though - I should be using the `EuiPageTemplate.EmptyPrompt`, etc, versions of the components.

There is a slight difference in the UI.  For instance, there is a bit more space at the top of this particular component which matches the previous implementation.

ex:
![image](https://github.com/elastic/kibana/assets/56395104/bc3d751b-b39a-47a1-9d5a-9cce766e3a62)

Reply:
Thanks for clarifying.